hosts/galileo/__init__.py

```diff
@@ -0,0 +1,395 @@
+    @staticmethod
+    def load_spks(t0, t1):
+        """Ensure that all the SPK kernels applicable near or within the time
+        interval tdb0 to tdb1 have been furnished.
+
+        The time can be tai or tdb.
+        """
+        Galileo.load_kernels(t0, t1, Galileo.SPK_LOADED, Galileo.SPK_LIST,
+                                     Galileo.SPK_DICT)
+
+    #===========================================================================
+    @staticmethod
+    def load_kernels(t0, t1, loaded, lists, kernel_dict):
+        from oops.unittester_support import TESTDATA_PARENT_DIRECTORY
+        import glob
+
+        kdir = os.path.join(TESTDATA_PARENT_DIRECTORY, '../SPICE/Galileo/')
+        gkdir = os.path.join(TESTDATA_PARENT_DIRECTORY, '../SPICE/General/')
+
+        cspyce.furnsh(gkdir + 'LSK/naif0012.tls')
+        cspyce.furnsh(kdir + 'SCLK/mk00062a.tsc')
+        cspyce.furnsh(kdir + 'IK/gll36001.ti')
+        cspyce.furnsh(kdir + 'SPK/de421.bsp')
+        cspyce.furnsh(kdir + 'SPK/de432s.bsp')
+
+        [cspyce.furnsh(ckfile) for ckfile in glob.glob(kdir + 'CK/*.bc')]
+        [cspyce.furnsh(spkfile) for spkfile in glob.glob(kdir + 'SPK/*.bsp')]
+
+
+        return
+
+## TODO:
+        # Find the range of months needed
+        m1 = int((t0 - Galileo.TDB0) // Galileo.DTDB)
+        m2 = int((t1 - Galileo.TDB0) // Galileo.DTDB) + 1
+
+        m1 = max(m1, 0)         # ignore time limits outside mission duration
+        m2 = min(m2, Galileo.MONTHS - 1)
+
+        # Load any months not already loaded
+        for m in range(m1, m2+1):
+          if not loaded[m]:
+            for kernel in lists[m]:
+                filespec = kernel.filespec
+                if filespec not in kernel_dict:
+                    spicedb.furnish_kernels([kernel])
+                    kernel_dict[filespec] = kernel
+                loaded[m] = True
+
+    ############################################################################
+    # Initialize the kernel lists
+    ############################################################################
+
+    @staticmethod
+    def initialize_kernels(kernels, lists):
+        """After initialization, lists[m] is a the KernelInfo objects needed
+        within the specified month.
+        """
+        for i in range(Galileo.MONTHS):
+            lists[i] = []
+
+        for kernel in kernels:
+
+            # Find the range of months applicable, extended by 12 hours
+            t0 = cspyce.str2et(kernel.start_time) - Galileo.SLOP
+            t1 = cspyce.str2et(kernel.stop_time)  + Galileo.SLOP
+
+            m1 = int((t0 - Galileo.TDB0) // Galileo.DTDB)
+            m2 = int((t1 - Galileo.TDB0) // Galileo.DTDB) + 1
+
+            m1 = max(m1, 0)     # ignore time limits outside mission duration
+            m2 = min(m2, Galileo.MONTHS - 1)
+
+            # Add this kernel to each month's list
+            for m in range(m1, m2+1):
+                lists[m] += [kernel]
```

Comment:
Maybe just use append?

Reply:
Is there a benefit?  This is how it's done in the cassini module.

Reply:
Just realized the isuue is append() appends a list as a single item, whereas + appends each item as a separate list item, which is the desired behavior.  In other words:

[1,2,3].append([4,5]) is [1,2,3, [4,5]], 

while

[1,2,3] + [4,5] is [1,2,3,4,5]

Reply:
Right, but `kernel` is a single object. `[kernel]` makes a list of length one with kernel as the only element. So these are equivalent:
```
lists[m] += [kernel]
lists[m].append(kernel)
```

Reply:
True, makes sense if appending a single object.